application_store_api/namespace_fund/data_store/store.py

```diff
@@ -0,0 +1,117 @@
+import datetime
+import json
+import uuid
+
+from application_store_api.namespace_fund.data_store.data import (
+    funding_service_design_application,
+)
+from application_store_api.namespace_fund.data_store.data import (
+    initial_fund_store_state,
+)
+from dateutil import parser as date_parser
+from dateutil.tz import UTC
+from slugify import slugify
+
+"""
+Data Access Object
+A data access object (DAO) is a pattern that provides an abstract interface
+to some type of database or other persistence mechanism.
+"""
+
+
+class application_data_access_object(object):
```

Comment:
Love using a design pattern to abstract away the fact we are using a dictionary instead of a real db (for now). 

You could go even further and create a DAO ABC which declares the functions we would expect from your DAO, that would be a fun exercise in creating design patterns in python and would enable future generations of devs to know what your api expects from your dao. 

This is not necessary though! but something to think about. (for fun)